build-system/tasks/build.js

```diff
@@ -74,6 +75,7 @@ async function doBuild(extraArgs = {}) {
     await compileCoreRuntime(options);
   } else {
     await compileAllJs(options);
+    await buildVendorConfigs(options);
```

Comment:
For consistency with `dist`, I think this should happen after `buildExtensions()`, and be gated by `if (!argv.core_runtime_only)`.